src/components/counter/counterDisplay.tsx

```diff
@@ -0,0 +1,28 @@
+import { Progress } from "antd";
+import * as React from "react";
+
+interface ICounterDisplayProps {
+    circleFormat: (percent: number) => string; // no idea wtf this is
```

Comment:
`:(percent: number) => string;` just means circle format is a function that takes a number param (percent) and returns a string.   

React composition is always an area of debate.  Passing this function in reduces the cohesiveness of the component, but makes it more flexible.  I typically preferrer cohesiveness over flexibility, until the flexibility is actually needed.

Reply:
did this one a little bit prematurely i guess :) got antsy with all the docs in the typescript deep dive and no coding application